spec/v1/annotations/object-model.yaml

```
@@ -25,9 +25,9 @@ definitions:
   "@ObjectModel.semanticKey":
     type: array
     description: |-
-      The property contains element(s) which shall be used to display the key in UIs (instead of the technical key).
+      The listed properties contains element(s) which shall be used to display the key in UIs (instead of the technical key).
```

```suggestion
      The entity contains element(s) which shall be used to display the key in UIs (instead of the technical key).
```

In case you accept the suggestion, also re-run generate.